test/test_point.py

```diff
@@ -107,6 +107,10 @@ def test_point_degrees_are_normalized(self):
         self.assertEqual((-85, -175, 375), tuple(point))
         point = Point(-95, -185, 375)
         self.assertEqual((85, 175, 375), tuple(point))
+        point = Point(-0.0, -0.0, 375)
+        self.assertEqual((0.0, 0.0, 375.0), tuple(point))
```

Comment:
I think we should explicitly check that the negative zeros are normalized to the positive ones, in order to avoid any peculiar bugs (like with [atan2](https://stackoverflow.com/a/4084219)).

The issue is that this assertion doesn't verify that, as `-0.0 == +0.0`:
```
>>> (-0.0, -0.0, 375) == (0.0, 0.0, 375.0)
True
```

I think that something like this will do:
```
    point = Point(-0.0, -0.0, 375)
    self.assertEqual((0.0, 0.0, 375.0), tuple(point))  # note that the zeros might be negative
    # ensure that negative zeros are normalized to the positive ones
    self.assertEqual((1.0, 1.0, 1.0), tuple(math.copysign(1.0, x) for x in point))
```